database/migrations/2023_05_08_143425_add_userid_to_dinnerform.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('dinnerforms', function (Blueprint $table) {
+            $table->integer('ordered_by')->nullable();
```

Comment:
Why is this nullable?

Reply:
It is nullable so all the existing dinnerforms do not default to user 0